python/src/pyraphtory/algorithms/twohoppaths.py

```diff
@@ -43,13 +43,14 @@ def iterate1(v: Vertex):
         return graph.step(step1).iterate(iterate1, 3, True)
 
     def tabularise(self, graph: TemporalGraph) -> Table:
-        def explode(r: Row):
-            return_rows = []
-            if len(r.get_values()) > 1:
-                if r.get(1) is not None:
-                    for hop_pair in r.get(1):
-                        return_rows.append(Row(r.get(0), *hop_pair))
-            else:
-                return_rows = []
-            return return_rows
-        return graph.select(lambda v: Row(v.name(), v[TWO_HOP_PATHS])).explode(explode)
+        # def explode(r: Row):
```

Comment:
Can probably delete

Reply:
theres no tests for this so wasn't sure if this actually works *emoji: this is fine*